Vagrantfile

```diff
@@ -6,36 +6,35 @@ sudo -i
 apt-get update
 
 # curl
-sudo apt-get install curl -y
+apt-get install curl -y
 
 # nodejs
-sudo apt-get -y install g++ git git-core nodejs npm
-sudo npm install n -g
-sudo n stable
+apt-get -y install g++ git git-core nodejs npm
+npm install n -g
+n stable
 node -v
 
 # PostgreSQL
-sudo apt-get install postgresql postgresql-contrib
-psql -u postgres postgres
+apt-get install postgresql postgresql-contrib -y
+sudo -u postgres psql -c "ALTER USER postgres PASSWORD 'postgres';"
```

Comment:
I think that this changes the Ubuntu user role from root to postgres. I couldn't seem to get it to work without it.